packages/cli/src/build.ts

```diff
@@ -149,6 +149,25 @@ function getInitialCompilerHost({
   );
 }
 
+/**
+ * Clean the output directory before building the project.
+ *
+ * @param project - The path to the project's `tsconfig.json` file.
+ * @param options - The compiler options to use.
+ * @param clean - Whether to clean the output directory before building.
+ */
+function cleanOutputDirectory(
+  project: string,
+  options: CompilerOptions,
+  clean = false,
+) {
+  const baseDirectory = dirname(project);
+  if (clean && options.outDir) {
+    info(`Cleaning output directory "${options.outDir}".`);
```

Comment:
Do we want to log for each project? Asking since we didn't log anything before.

Reply:
I intended for this to be logged when `verbose` is enabled, but somehow forgot that part. Fixed now.